test/e2e_node/critical_pod_test.go

```diff
@@ -50,6 +55,8 @@ var _ = SIGDescribe("CriticalPod [Serial] [Disruptive] [NodeFeature:CriticalPod]
 			}
 		})
 
+		podMap := make(map[string]*v1.Pod)
```

Comment:
if you want to put a global here, you should declare it here and initilize it in a BeforeEach block
```
var podMap map[string]*v1.Pod

ginkgo.BeforeEach(func() {
			ginkgo.By("Creating pods, starting with non-critical so that the critical preempts the other pods.")
			createdPods := f.PodClient().CreateBatch([]*v1.Pod{nonCriticalBestEffort, nonCriticalBurstable, nonCriticalGuaranteed})
	for _, pod := range createdPods {
		podMap[pod.Name] = pod
	}
}
```
xref https://onsi.github.io/ginkgo/#avoiding-test-pollution

Reply:
Thanks for pointing this out! learned something. Will change as suggested (not completely though, I'll elaborate in the next upload)